### DISCUSSION TITLE
Issues on YouTube

### DISCUSSION THREAD

**Discussion by NJC2** [sereneblue/chameleon#588] [Q&A]
I don't see any mention of this in the bugs sections, so I thought I'd ask here before submitting a bug report.  Starting about a week ago, I'm having to disable the add-on sometimes to use YouTube.  Videos will not play past 18-30 seconds when this addon is enabled.  This is not happening on every video, it seems random.  Even if I change profile and refresh it stops at the exact time, every time.  I can't even skip past the point where it stops, if I try, it won't play at all.  I haven't changed any settings, but I noticed the addon updated last week, which is right about the time this started happening.

**Comment by sereneblue** [comment]
Hi @NJC2, thanks for reporting. I am able to replicate this issue with any Chrome/Edge/Safari profile. I will continue to look into this because it is a bit incovenient.
